Sem. 04/Pract. 04/Task02.cpp

```diff
@@ -0,0 +1,142 @@
+#include <iostream>
+#include <fstream>
+#pragma warning (disable: 4996)
+
+const size_t MAX_SIZE_NAME = 25;
+const size_t MAX_SIZE_STATE = 100;
+const size_t NUM_OF_TASKS = 5;
+
+class Task {
+private:
+	char name[MAX_SIZE_NAME];
+	char state[MAX_SIZE_STATE];
+	int points;
+
+	bool isValidName(const char* name) {
+		if (name == nullptr)
+			return false;
+
+		if (strlen(name) >= MAX_SIZE_NAME)
+			return false;
+
+		return true;
+	}
+
+	bool isValidState(const char* state) {
+		if (state == nullptr)
```

Comment:
If statements aren't needed.

Add the conditions to a single return statement.